site/_scss/_global.scss

```diff
@@ -30,6 +30,8 @@
 
 :root {
   --flow-space: #{get-size(500)};
+  -moz-osx-font-smoothing: grayscale;
+  -webkit-font-smoothing: antialiased;
```

Comment:
Can we please rather set those on `body`? Using `:root` for actual properties and not only var declarations seems weird.

Reply:
I have fixed this in [this commit](https://github.com/GoogleChrome/developer.chrome.com/pull/7080/commits/f943fb03a2d1af33ef021e0f8fb58d14606fce77).